### ISSUE TITLE
Button in Aura theme does not meet WCAG AA standard

### ISSUE THREAD

**Issue by BeatsuDev** [primefaces/primeuix#169]
When using the Aura theme preset, the default background color for the Button component is `--p-primary-color` (#10b981). The text color is `--p-primary-contrast-color` (#ffffff). 

https://github.com/primefaces/primeuix/blob/db7eb95b51e49ab86e819131d411654f35a9e123/packages/themes/src/presets/aura/button/index.ts#L38-L52

This results in a [2.51:1 contrast ratio](https://webaim.org/resources/contrastchecker/?fcolor=FFFFFF&bcolor=10B981); way below the required WCAG AA 3.5:1 ratio, and directly contradicts the very first statement on the [PrimeVue Accessibility documentation page:](https://primevue.org/guides/accessibility/) "PrimeVue has WCAG 2.1 AA level compliance".

I understand the requirement for backwards compatibility as mentioned in the comment on #136, but since many companies are susceptible to receive fines for not complying with WCAG, I think either the docs should clearly specify that PrimeVue **does not** necessarily follow WCAG 2.1 AA level compliance for certain themes, or that this contrast ratio issue actually gets fixed even though it's a significant and potentially "breaking change".

**Comment by cagataycivici** [comment]
You can easily customize the colors per your needs to improve WCAG compliance per your needs.
